csatar-plugins/csatar/updates/builder_table_create_csatar_csatar_troops.php

```diff
@@ -0,0 +1,32 @@
+<?php namespace Csatar\Csatar\Updates;
+
+use Schema;
+use October\Rain\Database\Updates\Migration;
+
+class BuilderTableCreateCsatarCsatarTroops extends Migration
+{
+    public function up()
+    {
+        Schema::create('csatar_csatar_troops', function($table)
+        {
+            $table->engine = 'InnoDB';
+            $table->increments('id')->unsigned();
+            $table->timestamp('created_at')->nullable();
+            $table->timestamp('updated_at')->nullable();
+            $table->timestamp('deleted_at')->nullable();
+            $table->string('name', 255);
+            $table->string('email', 255)->nullable();
+            $table->string('website', 255)->nullable();
+            $table->string('facebook_page', 255)->nullable();
+            $table->string('troop_leader_name', 255);
+            $table->string('troop_leader_phone', 255);
+            $table->string('troop_leader_email', 255);
+            $table->integer('team_id')->unsigned();
```

Comment:
```suggestion
            $table->integer('team_id')->index('team_id')->unsigned();
```

Itt is későn szólok, de azt hiszem jobb lenne így. Kérlek a többit csináld így és az eddigieket én átírom, majd rollback-eljük és lefuttatjuk újra a migrációkat.